assets/stylesheets/components/_tables.scss

```diff
@@ -0,0 +1,105 @@
+// .table-header in _headings.scss
+
+table {
+  width: 100%;
+}
+
+.table-tagline {
+  border-top: 1px solid $table-border-color;
+  padding-top: .25rem;
+}
+
+.table-contrast-bg {
+  background: $table-bg;
+  padding: 1rem;
+}
+
+.receipt-table {
+  tr:not(:last-child) {
+    border-bottom: 1px solid $table-border-color;
+  }
+
+  tr:last-child {
+    td {
+      padding-bottom: 2rem;
+    }
+  }
+
+  tr {
+    td {
+      padding: .5rem 0;
+      vertical-align: top;
+
+      &.amount {
+        font-size: 1.5rem;
+      }
+    }
+
+    td:first-child {
+      @extend .text-uppercase;
+      font-size: .75rem;
+
+      a {
+        color: $cr-cyan;
+      }
+    }
+
+    td:last-child {
+      font-size: 1rem;
+      text-align: right;
+
+      &.amount {
+        font-size: 1.5rem;
+      }
+    }
+
+    small {
+      a {
+        color: $cr-cyan;
+      }
+    }
+  }
+}
+
+.streaming-table {
+  .new-day {
+    td {
+      padding-top: 2rem;
+    }
+
+    td:first-child {
+      font-weight: bold;
+    }
+  }
+}
+
+.dark-theme {
+  table,
```

Comment:
Similar feedback here. We shouldn't be stylizing all `table` elements. Please see my previous comment about using the `.table` selector instead.